app/tests/base_test.py

```diff
@@ -54,11 +54,12 @@ def get_error_msg(code):
                 return msg
             return 'Wrong status code'
 
-        self.assertEqual(code, response.status_code, msg=get_error_msg(response.status_code))
-        if code in [412, 304]:
+        code = code if isinstance(code, list) else [code]
+        self.assertIn(response.status_code, code, msg=get_error_msg(response.status_code))
```

Comment:
`codes = ` (plural) would be slightly more descriptive and indicate that it's a list

Reply:
Done